test/unit/com/unifina/service/StreamServiceSpec.groovy

```diff
@@ -179,7 +180,7 @@ class StreamServiceSpec extends Specification {
 		service.getReadAuthorizedStream("streamId", null, key, cb)
 		then:
 		thrown(NotPermittedException)
-		1 * service.permissionService.check(key, stream, Permission.Operation.STREAM_GET) >> false
+		1 * service.permissionService.check(key, stream, Permission.Operation.STREAM_SUBSCRIBE) >> false
```

Comment:
We need to think through what `getReadAuthorizedStream` is supposed to do. `STREAM_GET` seems correct, since the service method is called from the `show` action.